metals/src/main/scala/scala/meta/internal/metals/DoctorTargetInfo.scala

```diff
@@ -0,0 +1,10 @@
+package scala.meta.internal.metals
+
+final case class DoctorTargetInfo(
+    name: String,
+    scalaVersion: String,
+    definitionStatus: String,
```

Comment:
I propose to have an enum here for the status. I think sending emojis is not super friendly in the JSON API (if the client wants to displays something custom they would have to match on the emoji character).

Reply:
If not the emoji characters, what would the enum values be here?

Reply:
I would keep it simple and do:

```scala
sealed abstract class DoctorStatus
object DoctorStatus {
  case object Ok extends DoctorStatus
  case object Error extends DoctorStatus
}
```

and then serialize it as `"Ok"` and `"Error"` in the json response and as unicode characters in the html response.

Reply:
The use-case I'm assuming here is that the client won't display this value as-is but it will choose how to represent the information.

If we are sure clients will always be ok in displaying the emoji directly, I say it's fine to leave it as is

Reply:
Yea, that's a good point. For now, I know I'm going to just use it as is in `coc-metals`. My opinion is to leave it as if now, and later on of someone wants to display something custom based off those results, then we can easily change it then, but no need to now.

Reply:
Sounds good, let's see whether some other client uses this first 👍